Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -1123,6 +1123,51 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         },
     });
 
+    toObjectTest = new Y.Test.Case({
+        name: "eZ location Model toObject test",
+
+        setUp: function () {
+            this.loadResponse = {
```

Comment:
what's the point of adding a property to the test case if it's not used any where else than in this method. Just use a local variable